ui.go

```diff
@@ -43,15 +42,31 @@ func newUI(w fyne.Window, p fyne.Preferences, host func() string) *ui {
 }
 
 func (u *ui) askShutdown() {
-	dialog.ShowConfirm("Shutdown", "Are you sure you want to shut down?",
-		func(ok bool) {
-			if !ok {
-				return
-			}
-
-			cmd := exec.Command("shutdown", "-h", "now")
-			_ = cmd.Start()
-		}, u.win)
+	var pup *widget.PopUp
+	shutdown := &widget.Button{Text: "Power Off", Importance: widget.HighImportance, OnTapped: func() {
+		cmd := exec.Command("shutdown", "-h", "now")
+		_ = cmd.Start()
+	}}
+	reboot := &widget.Button{Text: "Reboot", OnTapped: func() {
+		cmd := exec.Command("shutdown", "-r", "now")
+		_ = cmd.Start()
+	}}
+	cancel := &widget.Button{Text: "Cancel", OnTapped: func() {
+		pup.Hide()
+	}}
+	spacer := canvas.NewRectangle(color.Transparent)
+	spacer.SetMinSize(fyne.NewSize(0, 10))
+	buttons := container.NewPadded(
+		container.NewVBox(
+			shutdown,
+			reboot,
+			spacer,
```

Comment:
These spacers kindof mess with the standard flow and look a little out of place IMHO

Reply:
I don't want my cancel button so close to my other options. A misclick is easiest when buttons are closest together.